src/extlibs/hrvo/agent_BACKUP_116425.h

```diff
@@ -0,0 +1,175 @@
+#pragma once
+
+#include "extlibs/hrvo/agent.h"
+#include "extlibs/hrvo/path.h"
+<<<<<<< HEAD
+#include "software/ai/navigator/path_planner/hrvo/velocity_obstacle.h"
+=======
+#include "extlibs/hrvo/velocity_obstacle.h"
+>>>>>>> 71803a43dad836407974703f97b6b3a106f0f097
+    #include "software/geom/vector.h"
```

Comment:
nit: There seems to be some merging left here

Reply:
not sure how these showed up but removed them